### PR TITLE
Fix failing tests

### PR DESCRIPTION
**Pull request by armandfardeau**
Fixes: 
https://github.com/decidim/decidim/pull/6143

- [x]  Make failing test pass
- [x]  Correct CSS mishaps (instructions bellow)

### CODE REVIEW

File: decidim-initiatives/config/locales/en.yml [hunk 1]
```diff
@@ -264,6 +264,8 @@ en:
             update: Update
           form:
             authorizations: Authorization settings
+            child_scope_threshold_enabled_help: This config flag doesn't support offline votes, it enables sub-jauges and works with an authorization handler that associate associates a scope to the user, make sure you select that authorization bellow in authorization settings. For more info on how this configuration works see this <a href="https://docs.decidim.org/admin-manual/en/initiatives/" target="_blank">link</a>. This is usually activated along together with "Only allow global scope initiative creation".
```
Comment (virgile-dev) [inline decidim-initiatives/config/locales/en.yml]:
```suggestion
            child_scope_threshold_enabled_help: This config flag doesn't support offline votes, it enables sub-jauges and works with an authorization handler that associates a scope to the user, make sure you select that authorization, bellow in authorization settings. For it to work scopes need to be configured in hierarchical way : 1 Parent - N Child. For more info on how this configuration works, see this <a href="https://docs.decidim.org/admin-manual/en/initiatives/" target="_blank">link</a>.
```

File: decidim-initiatives/config/locales/en.yml [hunk 2]
```diff
@@ -264,6 +264,8 @@ en:
...
+            only_global_scope_enabled_help: If you enable this make sure you configured your scopes and initiative type scopes properly, see this <a href="https://docs.decidim.org/admin-manual/en/initiatives/" target="_blank">link</a> to understand how. This is usually activated with "Enable child scope signatures". By enabling this scope selection will be skipped in initiative creation wizard
```
Comment (virgile-dev) [inline decidim-initiatives/config/locales/en.yml]:
```suggestion
            only_global_scope_enabled_help: Tick this flag if you enabled "Child scope signature" and configured the global scope as your parent scope. By enabling this, initiative type selection will be skipped in the initiative creation wizard. For more info on how this configuration works, see this <a href="https://docs.decidim.org/admin-manual/en/initiatives/" target="_blank">link</a>.
```

File: decidim-initiatives/config/locales/en.yml [hunk 3]
```diff
@@ -264,6 +264,8 @@ en:
             update: Update
           form:
             authorizations: Authorization settings
+            child_scope_threshold_enabled_help: "This config flag doesn't support offline votes, it enables sub-jauges and works with an authorization handler that associates a scope to the user, make sure you select that authorization, bellow in authorization settings. For it to work scopes need to be configured in hierarchical way : 1 Parent - N Child. For more info on how this configuration works, see this <a href=\"https://docs.decidim.org/admin-manual/en/initiatives/\" target=\"_blank\">link</a>."
+            only_global_scope_enabled_help: "Tick this flag if you enabled \"Child scope signature\" and configured the global scope as your parent scope. By enabling this, initiative type selection will be skipped in the initiative creation wizard. For more info on how this configuration works, see this <a href=\"https://docs.decidim.org/admin-manual/en/initiatives/\" target=\"_blank\">link</a>."
```
Comment (virgile-dev) [inline decidim-initiatives/config/locales/en.yml]:
```suggestion
            only_global_scope_enabled_help: "Tick this flag if you enabled \"Child scope signature\" and configured the global scope as your parent scope. By enabling this, the scope selection will be automatically made to Global scope in the initiative creation wizard and won't be displayed to the user. For more info on how this configuration works, see this <a href=\"https://docs.decidim.org/admin-manual/en/initiatives/\" target=\"_blank\">link</a>."
```

File: decidim-initiatives/config/locales/en.yml [hunk 4]
```diff
@@ -264,6 +264,8 @@ en:
             update: Update
           form:
             authorizations: Authorization settings
+            child_scope_threshold_enabled_help: 'This config flag doesn''t support offline votes, it enables sub-jauges and works with an authorization handler that associates a scope to the user, make sure you select that authorization, bellow in authorization settings. For it to work scopes need to be configured in hierarchical way : 1 Parent - N Child. For more info on how this configuration works, see this <a href="https://docs.decidim.org/admin-manual/en/initiatives/" target="_blank">link</a>.'
```
Comment (virgile-dev) [inline decidim-initiatives/config/locales/en.yml:267]:
```suggestion
            child_scope_threshold_enabled_help: 'This config flag doesn''t support offline votes, it enables sub-jauges and works with an authorization handler that associates a scope to the user, make sure you select that authorization, bellow in authorization settings. For it to work scopes need to be configured in hierarchical way : 1 Parent - N Child. For more info on how this configuration works, see this <a href="https://github.com/decidim/decidim/blob/develop/docs/advanced/initiatives_configs.md" target="_blank">link</a>.'
```

File: decidim-initiatives/config/locales/en.yml [hunk 5]
```diff
@@ -264,6 +264,8 @@ en:
...
+            only_global_scope_enabled_help: Tick this flag if you enabled "Child scope signature" and configured the global scope as your parent scope. By enabling this, initiative type selection will be skipped in the initiative creation wizard. For more info on how this configuration works, see this <a href="https://docs.decidim.org/admin-manual/en/initiatives/" target="_blank">link</a>.
```
Comment (virgile-dev) [inline decidim-initiatives/config/locales/en.yml:268]:
```suggestion
            only_global_scope_enabled_help: Tick this flag if you enabled "Child scope signature" and configured the global scope as your parent scope. By enabling this, initiative type selection will be skipped in the initiative creation wizard. For more info on how this configuration works, see this <a href="https://github.com/decidim/decidim/blob/develop/docs/advanced/initiatives_configs.md" target="_blank">link</a>.
```

**Comment by armandfardeau** [comment]
Thanks @ace

**Pull request merged**